test/main.hs

```diff
@@ -81,6 +81,17 @@ main = do
       unless (e1 == ExitSuccess && e2 == ExitSuccess)
             $ error "sleep exited with non-zero exit code!"
 
+    do
```

Comment:
Looks good! One last request: can you add a comment to the ChangeLog explaining the change in semantics here?